client/src/main/kotlin/app/cash/backfila/client/Connectors.kt

```diff
@@ -5,6 +5,10 @@ object Connectors {
   const val ENVOY = "ENVOY"
 }
 
+data class PerRunOverrideData(
+  val overrideClusterType: String? = null,
```

Comment:
Allow for choosing a target cluster type on a per-backfill-run basis, rather than always using the last-registered cluster type from the target service or a default.